build.gradle

```diff
@@ -1468,7 +1461,7 @@ project(':streams') {
       include('log4j*jar')
       include('*hamcrest*')
     }
-    from (configurations.runtime) {
+    from (configurations.runtimeClasspath) {
```

Comment:
We should also change `configurations.testRuntime` a few lines above to `configurations.testRuntimeClasspath`, but this causes a cyclic build error. Fixing this will be required before we move to Gradle 7.0.

Reply:
Is there a jira already?

Reply:
https://issues.apache.org/jira/browse/KAFKA-12417